spec/features/event_management_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+feature 'create an event' do
+  scenario 'has a new event page' do
+    visit new_event_path
+    expect(page).to have_content 'New event page'
+  end
+
+  feature 'add a new event with valid params' do
+    before(:each) do
+      @user = User.create!(username: 'test', email: 'test@example.com', password: 'password')
+      sign_in @user
+      visit new_event_path
+      fill_in 'Location', with: 'testing_location'
+      fill_in 'Event date', with: DateTime.now
+      click_on 'Create'
+    end
+
+    scenario 'redirects to root path after new event' do
+      expect(page).to have_content 'Upcoming Events'
```

Comment:
'Upcoming Events' is permanently present on the root path whether an event is created or not.
You should check for content that signifies that the event was created successfully instead.